src/node_api_jsrt.cc

```diff
@@ -443,7 +419,7 @@ static napi_status SetErrorCode(JsValueRef error,
     CHECK_JSRT(JsHasProperty(error, namePropId, &hasProp));
 
     JsValueRef nameValue = JS_INVALID_REFERENCE;
-    std::array<JsValueRef, 2> args = { nameArray, JS_INVALID_REFERENCE };
+    std::array<JsValueRef, 2> args = {{ nameArray, JS_INVALID_REFERENCE }};
```

Comment:
Would you mind explaining this? was the initializer list previously hitting a different constructor?

Reply:
clang (on macOS 10.13.4) specifically generated a warning that these braces should be doubled up.  Looking around it seems like most people suggest ignoring that warning and I see mention that clang 6.0 is specifically disabling the warning by default.

https://stackoverflow.com/questions/13905200/is-it-wise-to-ignore-gcc-clangs-wmissing-braces-warning

Given that I'll undo this change.